eslint-cjs-to-esm.js

```diff
@@ -21,10 +18,10 @@ try {
         }
     });
     if (process.env.DEBUG === "eslint-cjs-to-esm") {
-        console.debug({ args, eslintBin, builtinConfig });
+        console.debug({ args, builtinConfig });
     }
-    const { stdout, stderr } = await execa("node", [
-        eslintBin,
+    const { stdout, stderr } = await execa("npx", [
```

Comment:
I prefer to use `npm exec -- eslint <options>`
https://docs.npmjs.com/cli/v8/commands/npm-exec